go/Dockerfile-buster-1.13

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.13.15-buster
```

Comment:
Build.sh has some version hardcoded. We need to change this as well I suppose: https://github.com/fission/environments/blob/master/go/builder/build.sh#L25

Reply:
Yes, but thats for builder images